chia/full_node/mempool_check_conditions.py

```diff
@@ -132,40 +134,219 @@ def mempool_assert_my_amount(condition: ConditionWithArgs, unspent: CoinRecord)
     return None
 
 
-def get_name_puzzle_conditions(generator: BlockGenerator, max_cost: int, safe_mode: bool) -> NPCResult:
+def parse_aggsig(args: SExp) -> List[bytes]:
```

Comment:
maybe using consts for all the sizes we are validating would be nicer

Reply:
I looked into this, and I'm not sure it's worth it (at least not as part of this patch).

We have a name for the size of a public key (48) which is `G1Element.SIZE`. We would have to import that `blspy` module for that though.

We don't have a name for the max size of a message (1024). I don't think we have a name for the size of a SHA256 hash, other than perhaps `len(bytes32)`.

I suppose this file could introduce local names for those constants, is that what you meant?

Reply:
well yes, of course importing if its a dependency already is better, my opinion is it makes it easier to find things in the code, and usually if theres already a const it will encourage others to reuse 
this is very nitpicky though just thought it's worth mentioning